package/yast2-packager.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue May 10 12:31:13 UTC 2016 - igonzalezsosa@suse.com
+
+- Local repositories are disabled at the end of the installation
+  if they're not needed anymore (FATE#320494)
```

Comment:
The `not need need anymore` part is a bit vague and unclear, I'd write something like *if there is an online repository replacement available*  or something like that.